stores/users/store.go

```diff
@@ -0,0 +1,83 @@
+package users
+
+import (
+	"database/sql"
+	"errors"
+	"user-crud/models"
+	"user-crud/stores"
+)
+
+type userStore struct {
+	db *sql.DB
+}
+
+func New(db *sql.DB) stores.User {
+	return &userStore{db}
+}
+
+func (u *userStore) Insert(usr models.User) (models.User, error) {
+	query := "insert into user1 values(?,?,?,?,?)"
+	stmt, err := u.db.Prepare(query)
+	defer func() {
+		if err == nil {
+			stmt.Close()
+		}
+	}()
+	if err != nil {
+		return models.User{}, err
+	}
+	_, err = stmt.Exec(usr.Id, usr.Name, usr.Email, usr.Phone, usr.Age)
```

Comment:
do id autoincrement no need to insert id.
also after the insert operation fetches inserted record using getById and then returns that.